NightscoutServiceKit/OTPManager.swift

```diff
@@ -89,30 +89,32 @@ public class OTPManager {
         }
     }
     
-    public func validateOTP(otpToValidate: String) throws {
+    public func validatePassword(password: String, deliveryDate: Date) throws {
         
-        guard otpToValidate.count == passwordDigitCount else {
-            throw OTPValidationError.invalidFormat(otp: otpToValidate)
+        guard password.count == passwordDigitCount else {
+            throw OTPValidationError.invalidFormat(otp: password)
         }
         
```

Comment:
This will distinguish between a bad password and an expired. That should be helpful for user troubleshooting.